src/main/java/org/broadinstitute/hellbender/utils/hmm/segmentation/HiddenMarkovModelPostProcessor.java

```diff
@@ -109,7 +109,7 @@
      * Calculating variance posterior on long segments is time consuming; we assume hidden states become
      * independent if their separation is larger than the following value
      */
-    public static final double INDEPENDENT_TARGETS_SEPARATION_THRESHOLD = 1000;
+    public static final double INDEPENDENT_TARGETS_SEPARATION_THRESHOLD = 0;
```

Comment:
I guess so, as a reminder to what we are doing; at least until we switch to sampling for calculating segment CR variances. I have worked out the math a while ago. In principle, we must use `INDEPENDENT_TARGETS_SEPARATION_THRESHOLD = Double.POSITIVE_INFINITY` to get the exact segment variance in the current implementation but even `INDEPENDENT_TARGETS_SEPARATION_THRESHOLD = 1000` is too slow. So we set it to `0` now (i.e. use local CR marginal posteriors to estimate the segment variance) until we replace it with something better.